Demos/Blazorise.Demo/Pages/Tests/TreeViewPage.razor.cs

```diff
@@ -129,4 +130,11 @@ protected IEnumerable<T> Flatten<T>( IEnumerable<T> source, Func<T, IEnumerable<
                 queue.Enqueue( child );
         }
     }
+
+    protected Task OnNodeContextMenu( TreeViewNodeMouseEventArgs<NodeInfo> eventArgs )
```

Comment:
Should we have an actual menu example in the docs?

Reply:
Added an example in the docs.